pkg/object/metrics.go

```diff
@@ -28,6 +28,10 @@ var (
 		Help:    "Object requests latency distributions.",
 		Buckets: prometheus.ExponentialBuckets(0.01, 1.5, 20),
 	}, []string{"method"})
+	reqCount = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
The latency histogram already have a counter for the number of requests:
```
juicefs_object_request_durations_histogram_seconds_count{method="GET"} 1
juicefs_object_request_durations_histogram_seconds_count{method="PUT"} 1
```

Reply:
Oh ok, Thanks! The metric name does not make sense to me (durations? seconds? count?), but since it's already here...